pom.xml

```diff
@@ -106,6 +107,7 @@
         <rest-assured.version>4.1.1</rest-assured.version>
         <json-path.version>4.1.1</json-path.version>
         <jupiter.version>5.5.1</jupiter.version>
+
```

Comment:
Why do we need this new line?

Reply:
I can deleted it. Probably some remnant from refactor.